scripts/host-only/wkdev-sdk-bakery

```diff
@@ -69,7 +73,8 @@ deploy_image() {
 
     local image_directory="$(get_image_directory_by_name "${container_image_name}")"
     pushd "${image_directory}" &>/dev/null || _abort_ "Switching to directory '${image_directory}' failed"
-    run_podman_silent_unless_verbose push "$(get_tag_for_build)" || _abort_ "Pushing to registry failed"
+    run_podman_silent_unless_verbose push "$(get_tag_for_build)" || _abort_ "Pushing image to registry failed"
```

Comment:
I think you have to modify `get_tag_for_build` to include arch information. I don't think two images can have the same tag.

Reply:
Hmm, I thought that is not necessary since the information is in the manifests?

Reply:
```
$ podman manifest inspect ghcr.io/igalia/wkdev-sdk:multi-arch
{
    "schemaVersion": 2,
    "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json",
    "manifests": [
        {
            "mediaType": "application/vnd.oci.image.manifest.v1+json",
            "size": 2730,
            "digest": "sha256:fb24d36b418730f31d33693a9e3e077e15ad3421879789981b22fd700cfcbb23",
            "platform": {
                "architecture": "amd64",
                "os": "linux"
            }
        },
        {
            "mediaType": "application/vnd.oci.image.manifest.v1+json",
            "size": 2730,
            "digest": "sha256:084dff9e22c8f875278e0f55ce1085a0c7e9952c8a2f58662b4408038c9f4bed",
            "platform": {
                "architecture": "arm",
                "os": "linux",
                "variant": "v7"
            }
        }
    ]
}
```